.github/workflows/push.yml

```diff
@@ -124,18 +141,32 @@ jobs:
           arm64_set_path_and_python_version 3.9.7
           brew install autoconf automake libtool openssl pkg-config
           make --file ci/makefiles/osx.mk
-      - name: Build multi-arch apk Python 3 (armeabi-v7a, arm64-v8a, x86_64, x86)
+      - name: Build multi-arch sdl2 apk Python 3 (armeabi-v7a, arm64-v8a, x86_64, x86)
         run: |
           source ci/osx_ci.sh
           arm64_set_path_and_python_version 3.9.7
           make testapps-with-numpy
-      - name: Rename artifact to include the build platform name
+      - name: Rename sdl2 artifact to include the build platform name
+        run: |
+          mv testapps/on_device_unit_tests/${{ env.APK_ARTIFACT_FILENAME }} ${{ matrix.runs_on }}-${{ env.SDL2_APK_ARTIFACT_FILENAME }}
+      - name: Upload sdl2 apk artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: ${{ matrix.runs_on }}-${{ env.SDL2_APK_ARTIFACT_FILENAME }}
+          path: ${{ matrix.runs_on }}-${{ env.SDL2_APK_ARTIFACT_FILENAME }}
+      - name: Build multi-arch webview apk Python 3 (armeabi-v7a, arm64-v8a, x86_64, x86)
```

Comment:
Nice!

Minor: In order to reduce the build time, and avoid failures of the entire job if only one of the boostrap fails to build, how about using a `matrix` with `continue-on-error: true`?

As an example:
https://github.com/kivy/python-for-android/blob/4d45f99c22e027f695bb05c0c2d3f692b603d207/.github/workflows/push.yml#L227-L230

It will involve two runners instead of one, but may save us some debugging time in the future.

Reply:
That makes sense. Let me see if I can get that going.